sdk/README.md

```diff
@@ -207,11 +214,12 @@ https://github.com/LiskHQ/lisk-sdk/graphs/contributors
 
```

Comment:
**LINE 209**

**Note:** In case you face any issues during the installation, make sure you have the right version of `yarn` and `node` and try resetting the project with `yarn clean:full`.